clouddriver-azure/src/test/groovy/com/netflix/spinnaker/clouddriver/azure/resources/loadbalancer/deploy/ops/DeleteAzureLoadBalancerAtomicOperationUnitSpec.groovy

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2015 The original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.azure.resources.loadbalancer.deploy.ops
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.clouddriver.azure.security.AzureNamedAccountCredentials
+import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider
+import com.netflix.spinnaker.clouddriver.azure.resources.loadbalancer.ops.DeleteAzureLoadBalancerAtomicOperation
+import com.netflix.spinnaker.clouddriver.azure.resources.loadbalancer.ops.converters.DeleteAzureLoadBalancerAtomicOperationConverter
+import com.netflix.spinnaker.clouddriver.data.task.Task
+import com.netflix.spinnaker.clouddriver.data.task.TaskRepository
+import spock.lang.Shared
+import spock.lang.Specification
+
+class DeleteAzureLoadBalancerAtomicOperationUnitSpec extends Specification {
+  private static final LOAD_BALANCER_NAME = "azureapp1-st1-d1"
```

Comment:
Is the setup the same between this test and the previous test? If so extract the code into a single location.